numba/core/extending.py

```diff
@@ -309,25 +310,16 @@ def __repr__(self):
         return "<intrinsic {0}>".format(self._name)
 
     def __reduce__(self):
-        from numba.core import serialize
-
-        def reduce_func(fn):
-            gs = serialize._get_function_globals_for_reduction(fn)
-            return serialize._reduce_function(fn, gs)
-
         return (serialize._rebuild_reduction,
-                (self.__class__, str(self._uuid), self._name,
-                 reduce_func(self._defn)))
+                (self.__class__, str(self._uuid), self._name, self._defn))
 
     @classmethod
-    def _rebuild(cls, uuid, name, defn_reduced):
+    def _rebuild(cls, uuid, name, defn):
         from numba.core import serialize
 
         try:
             return cls._memo[uuid]
         except KeyError:
-            defn = serialize._rebuild_function(*defn_reduced)
-
```

Comment:
Note that this means that `rebuild_function` is no longer evaluated lazily, which may or may not matter.